src/arithmetic/rewrite_simplify.cc

```diff
@@ -140,6 +142,12 @@ class RewriteSimplifier::Impl : public IRMutator {
     if (dbound->max_value < val) {
       return kLT;
     }
+    if (dbound->min_value >= val) {
```

Comment:
dbound->min_value == val looks a bit cleaner?  We don't reach here when dbound->min_value > val.

Reply:
Good point, I didn't notice that. However I would leave it the way it is, because it feels more straightforward, and also it is more consistent with the case of symbolic bounds.